extensions/wikia/RTE/RTE.class.php

```diff
@@ -70,7 +73,7 @@ public static function replacePlaceholder($var) {
 	 *
 	 * @author: Macbre
 	 */
-	public static function renderPlaceholder($label, $data) {
+	public static function renderPlaceholder( $label, $data ) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$label". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1172)